watchtower/__init__.py

```diff
@@ -126,7 +126,7 @@ def __init__(
 
     def format(self, message):
         if self.add_log_record_attrs:
-            msg = message.msg if isinstance(message.msg, Mapping) else {"msg": message.msg}
+            msg = message.msg if isinstance(message.msg, Mapping) else {"msg": message.getMessage()}
```

Comment:
Do we know why this test case didn't catch the issue?:
https://github.com/kislyuk/watchtower/blob/e01bc29b24da3cb945203cb33668e2dde6c4eeb3/test/test.py#L293-L295

Reply:
I don't see how it would catch the issue. That test does not exercise any string formatting placeholders.

Reply:
Ahh, okay, I could not tell from this PR that was the issue. So we were just missing testing for this. Can this PR update that test (or add another) which includes formats within the string?